### PR TITLE
Retry loading items 3 times before giving up

### PR DESCRIPTION
**Pull request by gycn**

### CODE REVIEW

File: app/src/js/common/session_init.tsx [hunk 1]
```diff
@@ -249,24 +249,36 @@ function loadPointClouds (): void {
...
+        attemptsMap[sensorId] = 0
+        const onLoad = (geometry: THREE.BufferGeometry) => {
+          Session.pointClouds[item.index][sensorId] = geometry
 
-          () => null,
-
-          () => {
+          Session.dispatch(loadItem(item.index, sensorId))
+        }
+        const onError = () => {
```
Comment (fyu) [inline app/src/js/common/session_init.tsx]: Can we make finer error processing based on the ErrorEvent? https://github.com/mrdoob/three.js/blob/dev/src/loaders/FileLoader.d.ts#L17 Can we have the same error handling procedures for both images and point cloud?

Comment (gycn) [reply app/src/js/common/session_init.tsx]: Ok give me a few days to dig in to how the `ErrorEvent` fields are populated

Comment (gycn) [reply app/src/js/common/session_init.tsx]: From what I am seeing, the error is a `ProgressEvent`. It gives the status of the request. Is there any particular type of handling you want to check?
image

Comment (fyu) [reply app/src/js/common/session_init.tsx]: Are all the errors supposed to be tried repeatedly?

Comment (gycn) [reply app/src/js/common/session_init.tsx]: I think that for now we can just retry for all the errors. For point cloud, file formatting errors will not be caught in this section. I believe this is only for network errors. From what I have seen in the past, AWS will throw 404's sometimes even though the file exists. I can make another PR for the image loading later, but I would like this to be merged first for Dalu's use.

**Pull request merged**